mlir/lib/ExecutionEngine/APFloatWrappers.cpp

```diff
@@ -115,4 +115,16 @@ MLIR_APFLOAT_WRAPPERS_EXPORT uint64_t _mlir_apfloat_convert_to_int(
   val.convertToInteger(result, llvm::RoundingMode::NearestTiesToEven, &isExact);
   return result.getZExtValue();
 }
+
+MLIR_APFLOAT_WRAPPERS_EXPORT uint64_t _mlir_apfloat_convert_from_int(
```

Comment:
noob q: shouldn't we be returning a float? why bitcast back to int? is it so that we only need one wrapper?

Reply:
We chose to represent all floating-point values as `uint64_t` throughout `APFloatWrappers.cpp`. We can't return a float directly because there is no C++ type for `f4E2M1FN` etc. We also cannot return `APFloat` because it's not a primitive C++ type.